examples/first-frame/main.cpp

```diff
@@ -289,5 +289,18 @@ int main(int argc, char *argv[]) {
     LOG(INFO) << "Chip status error code: " << chipStatus;
     LOG(INFO) << "Imager status error code: " << imagerStatus;
 
+    //Example of reading temperature from frame metadata
+    uint32_t sensorTmpMetadata, laserTempMetadata;
+    status = frame.getTemperature(sensorTmpMetadata, laserTempMetadata);
+    if (status != Status::OK) {
```

Comment:
Can you inverse the logic?
if (status == Status::OK) {
  print the temperatures
}

Reply:
Fixed.